dockstore-webservice/src/test/java/io/swagger/model/ChecksumConverterTest.java

```diff
@@ -6,10 +6,20 @@
 import io.dockstore.webservice.core.Checksum;
 import io.dockstore.webservice.core.ChecksumConverter;
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.contrib.java.lang.system.SystemErrRule;
+import org.junit.contrib.java.lang.system.SystemOutRule;
 
 public class ChecksumConverterTest {
 
+    @Rule
+    public final SystemOutRule systemOutRule = new SystemOutRule().enableLog().muteForSuccessfulTests();
```

Comment:
Wonder if we should just extend something that has these two rules

Reply:
Could be worth a try, might not necessarily work though. 
It looks like `@Rule`  doesn't declare annotation inheritance, but junit might have implemented it another way